plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Tasks.js

```diff
@@ -253,9 +258,7 @@ Ext.define('NX.coreui.controller.Tasks', {
         list = me.getList();
 
     if (list) {
-      me.getTaskTypeStore().load(function() {
-        me.reselect();
-      });
+      me.getStore('TaskType').load();
```

Comment:
kinda sucky that we have to do this manually for every interaction, might be worthwhile to investigate switching to something more like this http://www.learnsomethings.com/2014/06/21/easy-data-editing-without-any-lines-of-code-using-autosync-true-in-extjs-5s-or-extjs4s-for-that-matter-grid/